RecoTracker/PixelSeeding/plugins/alpaka/CASimpleCell.h

```diff
@@ -0,0 +1,298 @@
+            continue;
+#ifdef CA_DEBUG
+          printf("Doublet no. %d %d doubletId: %ld -> %d (isKilled %d) (%d,%d) -> (%d,%d) %d %d\n",
+                 tmpNtuplet.size(),
+                 idx,
+                 doubletId,
+                 otherCell,
+                 cells[otherCell].isKilled(),
+                 this->inner_hit_id(),
+                 this->outer_hit_id(),
+                 cells[otherCell].inner_hit_id(),
+                 cells[otherCell].outer_hit_id(),
+                 idx,
+                 nInBin);
+#endif
+
+          last = false;
+          cells[otherCell].template find_ntuplets<DEPTH - 1>(acc,
+                                                             cc,
+                                                             cells,
+                                                             foundNtuplets,
+                                                             cellNeighborsHisto,
+                                                             cellTracksHisto,
+                                                             nCellTracks,
+                                                             ct,
+                                                             apc,
+                                                             quality,
+                                                             tmpNtuplet,
+                                                             minHitsPerNtuplet);
+        }
+        if (last) {  // if long enough save...
+          if ((unsigned int)(tmpNtuplet.size()) >= minHitsPerNtuplet - 1) {
+            {
+              hindex_type hits[TrackerTraits::maxDepth + 2];
+              auto nh = 0U;
+              constexpr int maxFB = 2;  // for the time being let's limit this
+              int nfb = 0;
+              for (auto c : tmpNtuplet) {
+                hits[nh++] = cells[c].theInnerHitId;
+                if (nfb < maxFB && cells[c].hasFishbone()) {
+                  ++nfb;
+                  hits[nh++] = cells[c].theFishboneId;  // Fishbone hit is always outer than inner hit
+                }
+              }
+              ALPAKA_ASSERT_ACC(nh < TrackerTraits::maxHitsOnTrack);
+              hits[nh] = theOuterHitId;
+              auto it = foundNtuplets.bulkFill(acc, apc, hits, nh + 1);
+#ifdef CA_DEBUG
+              printf("track n. %d nhits %d with cells: ", it, nh + 1);
+#endif
+              if (it >= 0) {  // if negative is overflow....
+                for (auto c : tmpNtuplet) {
+#ifdef CA_DEBUG
+                  printf("%d - ", c);
+#endif
+                  auto t_ind = alpaka::atomicAdd(acc, nCellTracks, (uint32_t)1, alpaka::hierarchy::Blocks{});
+
+                  if (t_ind >= uint32_t(ct.metadata().size())) {
+#ifdef CA_WARNINGS
+                    printf("Warning!!!! Too many cell->tracks associations (limit = %d)!\n", ct.metadata().size());
+#endif
+                    alpaka::atomicSub(acc, nCellTracks, (uint32_t)1, alpaka::hierarchy::Blocks{});
```

Comment:
just a suggestion, but
```suggestion
                    alpaka::atomicSub(acc, nCellTracks, 1u, alpaka::hierarchy::Blocks{});
```
would be more clear and compact :)